packages/core/src/data-grid/cells/text-cell.tsx

```diff
@@ -12,7 +12,7 @@ export const textCellRenderer: InternalCellRenderer<TextCell> = {
     needsHoverPosition: false,
     renderPrep: prepTextCell,
     useLabel: true,
-    render: a => drawTextCell(a, a.cell.displayData),
+    render: a => drawTextCell(a, a.cell.displayData, a.cell.contentAlign),
```

Comment:
I believe in this case only text cell would support content alignment. It would be useful to have this also activated on all the other cells using `drawTextCell` (especially number cell). The cells that currently use `drawTextCell`:
- number-cell
- markdown-cell
- row-id-cell
- text-cell
- uri-cell